tests/staticfiles_tests/storage.py

```diff
@@ -63,3 +63,17 @@ class SimpleCachedStaticFilesStorage(CachedStaticFilesStorage):
 
     def file_hash(self, name, content=None):
         return 'deploy12345'
+
+
+class MultiCachedStaticFilesStorage(CachedStaticFilesStorage):
+    """
+    A storage class to test pattern substitutions with more than one pattern
+    entry.
```

Comment:
I was thinking of also something like "The pattern rewrites strings like "url(...)" to JS_URL("...")" (I guess my main point is: can the regex be simplified at all?)